script.module.clouddrive.common/clouddrive/common/remote/errorreport.py

```diff
@@ -0,0 +1,29 @@
+#-------------------------------------------------------------------------------
+# Copyright (C) 2017 Carlos Guzman (cguZZman) carlosguzmang@protonmail.com
+# 
+# This file is part of Cloud Drive Common Module for Kodi
+# 
+# Cloud Drive Common Module for Kodi is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# Cloud Drive Common Module for Kodi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+
+import urllib
+
+from clouddrive.common.remote.request import Request
+
+
+class ErrorReport(object):
```

Comment:
Sending an error report over the net should be optional. Please replace with either a user prompt to send the data or a settings.xml option to enable reporting (default=disabled).

Reply:
It's optional. The setting is there in settings.xml ("report_error") and the check is in https://github.com/cguZZman/repo-scripts/blob/8da551458c3a7b6ff65345d3c04647c2e346deeb/script.module.clouddrive.common/clouddrive/common/ui/addon.py#L539
It's enabled by default though. 
I'll move the check to the ErrorReport class and make it disabled by default.

Reply:
Great, I must have missed it...